### ISSUE TITLE
Inconsistent menu names

### ISSUE THREAD

**Issue by digiajay** [JKISoftware/JKI-State-Machine-Editor#40]
In tree window menu it's named as "Frames"
In case structure menu  it's named as "States"
![screenshot 28-06-2017 154042](https://user-images.githubusercontent.com/6551395/27632272-d8e05f84-5c18-11e7-8da0-6a9741518446.png)

**Comment by jimkring** [comment]
You're right that this is inconsistent -- I don't really like it, either.

The reason (and it's not a great reason) is that the Tree View can operate on the Event Structure and "Frame" applies to both Case Structure and Event Structure "Frames" whereas the right-click options on the Case Structure and on the State Strings are really always "State" operations.
